modules/serverless-connect-func-source-odc.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-eventing.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-connect-func-source-odc_{context}"]
+= Connect an event source to a function using the Developer perspective
+
+Functions are deployed as Knative services on an {product-title} cluster. When you create an event source by using the {product-title} web console, you can specify a deployed function that events are sent to from that source.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console and are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created and deployed a function.
+
+.Procedure
+
+. Create an event source of any type, by navigating to *+Add* -> *Event Source* and then selecting the event source type that you want to create.
+
+. In the *Sink* section of the *Create Event Source* form view, select your function in the *Resource* list.
+
+. Click *Create*.
+
+.Verification
```

Comment:
It is also possible to verify from the local CLI using `func invoke` but that may be more than you want to get into here. Just noting it...

Reply:
+1 for out of scope; this section is specific for ODC